### PR TITLE
Second Task

### PR DESCRIPTION
**Pull request by babinskiy**

### CODE REVIEW

File: 02_sum_range.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+import argparse
+
+def main(start, end):
+    None
```
Comment (mike-unitskyi) [inline 02_sum_range.py]: is this really needed?

File: 02_sum_range.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+import argparse
```
Comment (mike-unitskyi) [inline 02_sum_range.py]: argparse isn't builtin python module, would like to see `README.md` file with install and run documentation.

File: 02_sum_range.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+def main(start, end):
+    None
+    numbers = range(start, end)
```
Comment (mike-unitskyi) [inline 02_sum_range.py]: range isn't closed (it doesn't take into account end value)

File: 02_sum_range.py (hunk 1 above)
Comment (babinskiy) [reply 02_sum_range.py]: :)  remains from subs snippet

File: 02_sum_range.py (hunk 2 above)
Comment (babinskiy) [reply 02_sum_range.py]: Fixed

File: 02_sum_range.py (hunk 1 above)
Comment (babinskiy) [reply 02_sum_range.py]: Fixed

File: 02_sum_range.py (hunk 3 above)
Comment (babinskiy) [reply 02_sum_range.py]: It wasn't mentioned in task...
Fixed

**Pull request merged**